Sources/Peasy/Server.swift

```diff
@@ -14,6 +14,13 @@ import Foundation
 public final class Server {
 	
 	// MARK: - Properties -
+	// MARK: Public
+	
+	/// The port that the server has been started on
+	///
+	/// Returns nil if the server hasn't been started or has been stopped
+	public var port: Int?
```

Comment:
Instead of storing it as a property could we not just return it at the end of the `start` function?

Reply:
I don't know if either of these are actually necessary because the code looping through the ports already knows the port that was successful because the try passes (you'd need to modify your loop a bit potentially)

Reply:
Yep I previously had it so it was remembering the port and then checking if the state was running, but it was tidier like this and it seemed like something the server could store.
But I can understand wanting to keeping the API surface area as small as possible